cli.js

```diff
@@ -96,6 +97,10 @@ function run(file) {
 		args.push('--fail-fast');
 	}
 
+	if (cli.flags.serial) {
+		args.push('--serial');
+	}
```

Comment:
Instead of doing this for every option I think we should just JSON.stringify the options and pass them as the first argument.